utils/dev-tool/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "lockbook-dev"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+structopt = "0.3.13"
+port_scanner = "0.1.5"
```

Comment:
we probably don't want this, just pick a random number and later we'll have logic in there to have the server restart if it fails to start

Reply:
between 1024-u16::max_value